green/cmdline.py

```diff
@@ -68,6 +71,12 @@ def main(argv=None, testing=False):
     # Actually run the test_suite
     result = run(test_suite, stream, args, testing)
 
+    # Generate a test report if required
+    if args.junit_report:
```

Comment:
This the only code you did not execute in your tests (which I greatly appreciate -- many contributions come without any tests at all)!  Let's keep up our 100% test coverage!  Make one more test in `green/test/test_cmdline.py`, please! 😉

Reply:
You're right. I just added a new test case in the `test_cmdline.py` module.